src/components/ui/icons/SpeakerLayout/index.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import Svg, { SvgProps } from '../Svg';
+
+const SpeakerLayout: React.SFC<SvgProps> = (props) => (
+  <Svg {...props}>
+    <path d="M7.793 15.34c.553 0 1 .448 1 1v1.566c0 .552-.447 1-1 1H6c-.552 0-1-.448-1-1v-1.565c0-.553.448-1 1-1h1.793zm10.207 0c.552 0 1 .448 1 1v1.566c0 .552-.448 1-1 1h-1.793c-.553 0-1-.448-1-1v-1.565c0-.553.447-1 1-1H18zm-5.103 0c.552 0 1 .448 1 1v1.566c0 .552-.448 1-1 1h-1.794c-.552 0-1-.448-1-1v-1.565c0-.553.448-1 1-1h1.794zm-5.104 1H6v1.566h1.793v-1.565zm10.207 0h-1.793v1.566H18v-1.565zm-5.103 0h-1.794v1.566h1.794v-1.565zM17 5.095c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2H7c-1.105 0-2-.895-2-2v-5c0-1.105.895-2 2-2h10zm0 1H7c-.513 0-.936.386-.993.883L6 7.094v5c0 .513.386.936.883.993l.117.007h10c.513 0 .936-.386.993-.883l.007-.117v-5c0-.513-.386-.935-.883-.993L17 6.094z" />
+  </Svg >
+);
+
+SpeakerLayout.displayName = "SpeakerLayout";
```

Comment:
Can we make this single quotes I see the newly few added icons have this double quote but rest all are single quotes?